### PR TITLE
pkg/orchestrator(ticdc): migrate test-infra to testify for pkg/orchestrator/util pkg

### LINKED ISSUE #4878: migrate test-infra to testify for `pkg/orchestrator/util` pkg

### PR DESCRIPTION
**Pull request by mikechengwei**
### What problem does this PR solve?

Issue Number: close #4878 .

### What is changed and how it works?
This pr migrate test-infra to testify for pkg/orchestrator/util pkg.

### Check List 

Tests 

 - Unit test

Code changes

None

Side effects

None

Related changes
None

### Release note 

```release-note
None
```

### CODE REVIEW

File: pkg/orchestrator/util/key_utils_test.go [hunk 1]
```diff
@@ -14,80 +14,66 @@
 package util
 
 import (
+	"github.com/stretchr/testify/require"
 	"testing"
```
Comment (hi-rustin) [inline pkg/orchestrator/util/key_utils_test.go:18]:
```suggestion
    "testing"
    
	"github.com/stretchr/testify/require"
```

**Review by hi-rustin — APPROVED** [review]
rest LGTM

**Comment by zhaoxinyu** [comment]
@mikechengwei , some fmt problems are detected in the lint procedure. Please execute `make check` and see if any fmt problems can happen. BTW, `make check` can automatically fix the fmt problems for you.

**Review by overvenus — APPROVED** [review]

**Comment by overvenus** [comment]
> @mikechengwei , some fmt problems are detected in the lint procedure. Please execute `make check` and see if any fmt problems can happen. BTW, `make check` can automatically fix the fmt problems for you.

/hold

**Comment by mikechengwei** [comment]
> @mikechengwei , some fmt problems are detected in the lint procedure. Please execute `make check` and see if any fmt problems can happen. BTW, `make check` can automatically fix the fmt problems for you.

Thanks for remind, i have fix the format code error.

**Pull request merged**